### ISSUE TITLE
Add functionality to use different labels

### ISSUE THREAD

**Issue by collinmcneese** [jmassardo/issues-to-totango#83]
Currently this action keys off of `task` and `touchpoint` labels for functions.  These labels should be user-configurable to provide an alternate name, if desired, in the event that these label names are too ambiguous or do not work in a given repository.

for example: allow a user to specify that `ttask` should be the label to use for matching the `task` actions in this workflow.

**Comment by frye** [comment]
I wonder if we could extend this a bit. At the moment the touchpoint labels are configured as secrets. If we parsed that information from labels we could manage different types of touchpoints based on issue labels. And we could use the secret as a fall back / default.

**Comment by collinmcneese** [comment]
Yeah, I think having another issue to make `TOUCHPOINT_TAGS` dynamic or default would be nice as well
